src/Microsoft.AspNet.Mvc.Razor.Host/MvcRazorHost.cs

```diff
@@ -45,18 +45,18 @@ public class MvcRazorHost : RazorEngineHost, IMvcRazorHost
         /// </summary>
         /// <param name="root">The path to the application base.</param>
         public MvcRazorHost(string root) :
```

Comment:
We should probably change the consumer of this to use the new property `hostingEnvironment.WebRootFileProvider`